File: AdvancedScada.Controls/HslControls/Keypad/IKeyboard.cs
using System.Drawing;
namespace AdvancedScada.Controls.HslControls.Keypad
{
    public delegate void ButtonClickEventHandler(object sender, KeypadEventArgs e);

    public interface IKeyboard
    {
        // Events
        event ButtonClickEventHandler ButtonClick;

        // Properties
        Font Font { get; set; }
        Color ForeColor { get; set; }
        Point Location { get; set; }
        object StartPosition { get; set; }
        string Text { get; set; }
        bool TopMost { get; set; }
        string Value { get; set; }
        bool Visible { get; set; }

    }





}

File: AdvancedHMI.Controls_Net45/Keyboard/IKeyboard.vb
Imports System.Drawing

Public Interface IKeyboard
    ' Events
    Event ButtonClick As ButtonClickEventHandler

    ' Properties
    Property Font As Font
    Property ForeColor As Color
    Property Location As Point
    Property StartPosition As Object
    Property [Text] As String
    Property TopMost As Boolean
    Property Value As String
    Property Visible As Boolean

    ' Nested Types
    Delegate Sub ButtonClickEventHandler(ByVal sender As Object, ByVal e As KeypadEventArgs)
End Interface




